Teradata/covalent-quickstart
# Update Test Harness

## Description

After writing a number of tests, the stock karma configuration from angular/cli isn't the greatest.

I've added a number of configuration changes that should help with reading test failure output ( shortened backtraces ), seeing test coverage stats and adding a `yarn test:watch` to the package.json scripts for continuous run for development.

Also fixed a broken test in the overview component.

#### Test Steps
`yarn start` see the app still works
`yarn test` see the new test output and tests pass

<img width="965" alt="screen shot 2017-08-24 at 8 37 42 pm" src="https://user-images.githubusercontent.com/239773/29695087-9f421c4a-890e-11e7-867c-29f3ed56720a.png">
